spec/features/events_spec.rb

```diff
@@ -145,6 +143,19 @@
     end
   end
 
+  scenario "logged in as Organizer I can lock the event application statuses by pressing one of the email buttons" do
+    login(:organizer)
+    @pupil = FactoryGirl.create(:profile)
+    @application_letter = FactoryGirl.create(:application_letter_accepted, event: @event, user: @pupil.user)
+    @application_letter.save
```

Comment:
If I'm not mistaken the save was only to save the new status, which is now already set in the factory